pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-H10GiPyAvX6UVM5by4TqW+z6tcwPqskMnTML3BWJdVU=";
+  };
+
+  postPatch = ''
+    touch requirements.txt
+  '';
+
+  build-system = with python3Packages; [ setuptools ];
+
+  dependencies = with python3Packages; [
+    cachecontrol
+    lxml-html-clean
+    requests
+    six
+  ];
+
+  meta = {
+    description = "Python package for cloning complete webpages and websites to local storage";
+    homepage = "https://github.com/rajatomar788/pywebcopy/";
+    license = lib.licenses.asl20;
+    mainProgram = "pywebcopy";
+    maintainers = with lib.maintainers; [ d3vil0p3r ];
+    platforms = lib.platforms.unix;
```

Comment:
you can ommit this, it will use default platforms for python

Reply:
Removed `platforms`.